tests/BUILD

```diff
@@ -82,6 +82,7 @@ cc_test(
     ],
 )
 
+"""
```

Comment:
Why?

Reply:
I have openede another issue #325 to solve this issue.
Test for MAM related API is relative hard to implement.

Reply:
Shall we wait for another PR?

Reply:
I think we don't need to wait for that PR, since the tests of `map` would definitely take some time to finish.
Therefore, I personally would prefer to implement both MAM send and receive APIs first, then working on the tests.

Reply:
Avoid preserving dead code without explicit reasons.

Reply:
I intended to make minimal changes. So, please don't turn off tests without explicit reasons. You shall append something meaningful and traceable.

Reply:
But all the tests in `test_map_mode.c` are out of date. Therefore, I have to choose delete the `test_map_mode.c` or delete the bazel run to test it

Reply:
Let's clean up.